### DISCUSSION TITLE
How to use Modal API at  editor.DomComponents.addType

### DISCUSSION THREAD

**Discussion by PanuwatChinpratan** [GrapesJS/grapesjs#5623] [Q&A] [answered]
Need trigger Open Modal at  editor.DomComponents.addType

**Comment by javadsamiee** [comment, marked as answer]
`editor.Modal.open({ title: 'you title', content: '' })`

**Reply by PanuwatChinpratan** [reply]
Exam pls

**Reply by javadsamiee** [reply]
I added a "Sample block", you can see modal after adding it

https://codepen.io/jsnotjs/pen/yLwbPPE?editors=0010

**Reply by PanuwatChinpratan** [reply]
Thank your sir

**Comment by THEWebDegenerate** [comment]
$hh
